web/templates/_base.html

```diff
@@ -18,153 +18,77 @@
-      border: solid 1px grey;
-      }
-      div.transaction.type-Status {
-      background: #FFE;
-      }
-
-      div.theader {
-      width: 98%;
-      background: #BBB;
-      display: inline-flex;
-      flex-direction: row;
-      flex-wrap: wrap;
-      margin-bottom: 10px;
-      padding: 3px;
-      }
-      div.theader div {
-      padding-right: 10px;
-      }
-      div.theader .email {
-      font-weight: 800;
-      }
-
-      /* Page stuff */
-      div#pheader {
-      width: 98%;
-      padding: 5px;
-      background: blue;
-      color: white;
-      font-weight: 900;
-      margin-bottom: 1em;
-      clear: both;
-      height: 3em;
-      }
-
-      .box#pheader2 {
-        border-left: solid 6px blue;
-        font-weight: 900;
-        font-size: 150%;
-        width:98%;
-      }
-
-      div#pfooter {
-      width: 98%;
-      padding: 5px;
-      background: blue;
-      color: white;
-      margin-top: 1em;
-      clear:both;
-      min-height: 3em;
-      }
-
-      a.button {
-        background: buttonface;
-        text-decoration: none;
-        border: solid 1px #999;
-        color: black;
-        padding: 4px;
-      }
-ul.tickets {
-  list-style-type: none;
-  padding-left: 2em;
-  text-indent: -2em;
-}
-      li.ticket {
-      }
-      .ticket a {
-        color: black;
-        text-decoration:none;
-      }
-      .ticket a:hover {
-        color: #777777;
-      }
-
-      .box#pfooter2 {
-        border-left: solid 6px green;
-        width:98%;
-      }
-    </style>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+      <meta name="description" content="">
+      <meta name="author" content="">
+      <link rel="icon" href="/static/favicon.ico">
+
+      <title>{{ template "Title" . }} | Perlbug Archive</title>
+
+      <link rel="canonical" href="https://getbootstrap.com/docs/4.0/examples/jumbotron/">
+
+      <!-- Bootstrap core CSS -->
+      <link href="/static/vendor/bootstrap/bootstrap.min.css" rel="stylesheet">
+
+     <!-- Vendor CSS-->
+     <link rel="stylesheet" href="/static/vendor/fontawesome/css/all.min.css"> <!--load all styles -->
+
+      <!-- Custom styles for this template -->
+      <link href="/static/app/jumbotron.css" rel="stylesheet">
+
     <meta name="robots" content="noindex, nofollow">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <title>{{ template "Title" . }} | Perlbug Archive</title>
+    <style type="text/css">
+        div.content {
+            margin-left: 1em;
+            margin-right:1em;
+            font-family: monospace;
+            white-space: pre;
+        }
+    </style>
+
   </head>
   <body>
-    <div class="box" id="pheader2">
-      Perlbug Archive: {{ .Site }}
-    </div>
+
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+      <a class="navbar-brand" href="/">Static Perl 5 RT {{ .Site }}</a>
```

Comment:
/ isn't right ,you need to use the value of the prefix.

Reply:
yeap yeap lol going to update it, are we really going to use this prefix?

Reply:
`{{$Prefix}}`